Wireframe/index.html

```diff
@@ -4,10 +4,63 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Wireframe</title>
-    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="css/style.css">
   </head>
   <body>
-    <!-- Add your HTML markup here -->
-    <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
-</body>
-</html
\ No newline at end of file
+    <header id="hero">
+      <h1> Git for Developers</h1>
+      <p>A Versatile Collaberation Tool </p>
+    </header>
+    <main>
+      <section>
+        <article id="git">
```

Comment:
Your other two article elements have a class.  Should this one also have a class to be consistent?
The layout does  behave nicely when you resize the window.